CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
+1. Change your `nunjucks.configure` property so it only includes the  `node_modules/govuk-frontend/` directory:
 
-  For example the accordion component selector has changed from `[data-module="accordion"]` to `[data-module="govuk-accordion"]`.
-
-  ```html
-  <div class="govuk-accordion" data-module="govuk-accordion">
-    <!-- ... -->
-  </div>
-  ```
+```nunjucks
+nunjucks.configure([
+  "node_modules/govuk-frontend/"
+])
+```
 
-  If you are initialising components manually, you will need to add the `govuk-` prefix to any CSS selectors used to find GOV.UK Frontend components:
+2. If you've extended the [page template](https://design-system.service.gov.uk/styles/page-template/), add `govuk/` to the template path:
 
-  ```javascript
-  var $accordions = document.querySelector('[data-module="govuk-accordion"]')
-  ```
+```nunjucks
+{% extends "govuk/template.njk" %}
+```
 
-  ([PR #1443](https://github.com/alphagov/govuk-frontend/pull/1443))
+3. Update the import paths in your components so they include `govuk/components/`. For example:
 
-- Button and details components are now initialised consistently
+```nunjucks
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+```
 
-  The button and details components now use the `[data-module]` selector to initialise them.
+#### Update asset paths
 
-  If you're using Nunjucks macros, and are [using the `initAll` function](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) you will not be affected.
+Update the assets path to include the `govuk/` directory:
 
-  ### Button component
+`/node_modules/govuk-frontend/govuk/assets`
 
-  The button component now is initialised with the `[data-module="govuk-button"]` selector instead of globally based on the document.
+If your project uses Express.js, you must also use the following code in your configuration file:
 
-  If you are using HTML, you need to add the `[data-module="govuk-button"]` attribute to your buttons:
+```javascript
+app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
+```
+Pull requests:
 
-  ```html
-  <button ... data-module="govuk-button">...</button>
-  ```
+- [#1458: Namespace nunjucks and components](https://github.com/alphagov/govuk-frontend/pull/1458)
+- [#1467: Update the main entry point in package.json](https://github.com/alphagov/govuk-frontend/pull/1467)
 
-  If you are initialising the button component manually, you will need to reference this `[data-module]` attribute:
+### Migrate to the new accessible focus states
 
-  ```javascript
-  var $buttons = document.querySelector('[data-module="govuk-button"]')
-  ```
+The focus state of components now meets the new WCAG 2.1 level AA requirements.
 
-  ### Details component
+You must [update your component’s focus state](xxxx) to make your design consistent with our new focus styles.
```

Comment:
Completely non-blocking: "our" sounds kind of proprietary. Maybe "GOV.UK Frontend focus styles"